gateway/utils.py

```diff
@@ -70,9 +70,8 @@ def get_swagger_from_url(api_url: str):
     """
     try:
         return requests.get(api_url).json()
-    except requests.exceptions.ConnectionError as error:
-        raise requests.exceptions.ConnectionError(
-            f'Please, check that {api_url} is accessible.') from error
+    except requests.exceptions.ConnectionError:
+        raise ConnectionError
```

Comment:
It's better to keep the previous error message instead of raising an error without any.